api/v1/routes/auth.py

```diff
@@ -0,0 +1,89 @@
+from fastapi import FastAPI, Depends, HTTPException, status, APIRouter
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from sqlalchemy.orm import Session
+from pydantic import BaseModel
+from typing import Optional
+from ..models.user import User
+from ..models.profile import Profile
+from datetime import datetime, timedelta
+from api.v1.schemas.token import Token, LoginRequest
+from api.v1.schemas.auth import UserCreate
+from api.utils import json_response
+from api.db.database import get_db
+from api.utils.auth import authenticate_user, create_access_token, get_current_admin, get_current_user,hash_password,get_user
+
+
+from api.v1.models.org import Organization
+
+from api.v1.models.product import Product
+
+
+db = next(get_db())
+
+
+router = APIRouter()
+
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
+
+@router.post("/login", response_model=Token,status_code=status.HTTP_200_OK)
```

Comment:
Give the router a proper semantic name, and add a `/auth` prefix and `auth` tag for swagger docs. to the `APIRouter` class

Also create a base router for `/api/v1`, and include other routers to it

Reply:
i dont know what you mean here